assets/debug.go

```diff
@@ -34,7 +34,7 @@ type asset struct {
 
 // templatesErrorTmpl reads file data from disk. It returns an error on failure.
 func templatesErrorTmpl() (*asset, error) {
-	path := "/Users/chambr/go/src/github.com/ONSdigital/dp-frontend-router/assets/templates/error.tmpl"
+	path := "/Users/iankent/dev/src/github.com/ONSdigital/dp-frontend-router/assets/templates/error.tmpl"
```

Comment:
should this file be committed?

Reply:
it always has been, but it probably doesn't need to be - it gets rebuilt when running `make debug` anyway

I won't remove it as part of this just in case it does somehow break something, I'll open another PR to do it later!